javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,47 @@
+//GIPHY
+document.addEventListener("DOMContentLoaded", giphyInit);
+
+function giphyInit() {
+  const btnSearch = document.querySelector(".btnSearch");
+  btnSearch.addEventListener("click", (ev) => {
+    ev.preventDefault(); // to stop the page reload
+    let numGif = document.querySelector(".num").value;
+    console.log("num:" + numGif);
+
+    let url = `http://api.giphy.com/v1/gifs/search?q=ryan+gosling&api_key=FtR8NF0BZTC8WREmYfUfvQYzxO8uGHBA&limit=${numGif}&q=`;
+
+    let searchInput = document.querySelector(".search").value.trim();
+
+    url = url.concat(searchInput); // concat whatever they tayped inside the form
```

Comment:
I'm not sure why you are adding each gif searched to the previous one. Here with concating the searchInput, you are actually adding new gifs to the previous gifs to be shown in your new search.